package.json

```diff
@@ -39,11 +40,15 @@
         "@eslint/js": "^9.8.0",
         "@lwc/eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
         "@lwc/test-utils-lwc-internals": "link:./scripts/test-utils",
+        "@nx/js": "19.5.6",
```

Comment:
Tildes vs carets vs bare version numbers are all over the place in this repo, but it doesn't really matter since it's `devDep`s and so the lockfile is the truth.